fb_instant_articles.module

```diff
@@ -354,60 +333,161 @@ function fb_instant_articles_fb_instant_articles_transformer_rules() {
       ),
     ),
     array(
-      'class' => 'SocialEmbedRule',
+      'class' => 'H2Rule',
+      'selector' => 'h3,h4,h5,h6',
+    ),
+    array(
+      'class' => 'InteractiveRule',
+      'selector' => 'blockquote.instagram-media',
```

Comment:
This is a rule that you should implement in a custom module, Instagram embeds are not universal.